minecraft/dial.go

```
@@ -194,31 +205,56 @@ func (d Dialer) DialContext(ctx context.Context, network, address string) (conn
+		if d.XBLClient == nil {
+			x, ok := d.TokenSource.(xsapi.TokenSource)
+			if !ok {
+				x = auth.ContextSession(ctx, d.TokenSource)
+			}
+			client, err := xsapi.ClientConfig{
+				HTTPClient: d.HTTPClient,
+			}.New(ctx, x)
+			if err != nil {
+				return nil, &net.OpError{Op: "dial", Net: "minecraft", Err: fmt.Errorf("login to xbox live: %w", err)}
+			}
+			defer client.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `client.Close` is not checked

(errcheck)